package.json

```diff
@@ -242,7 +242,7 @@
     "fix": "eslint --ext .js,.jsx,.tsx,.ts . --quiet --fix --cache",
     "build": "cross-env NODE_ENV=production webpack",
     "run": "webpack --progress --watch",
-    "dev-server": "webpack-dev-server -d",
+    "dev-server": "webpack serve",
```

Comment:
Probably we want to pass `--env development --mode development`?

Reply:
Good call. I'll add that